docs/sphinx/source/whatsnew/v0.6.0.rst

```diff
@@ -11,9 +11,37 @@ API Changes
 
 Enhancements
 ~~~~~~~~~~~~
-* Add sea surface albedo in irradiance.py (:issue:`458`)
-* Implement first_solar_spectral_loss in modelchain.py (:issue:`359`)
-* Clarify arguments Egref and dEgdT for calcparams_desoto (:issue:`462`)
+* Add sea surface albedo in ``irradiance.py`` (:issue:`458`)
+* Implement :meth:`~pvlib.modelchain.ModelChain.first_solar_spectral_loss`
+  in ``modelchain.py`` (:issue:`359`)
+* Clarify arguments ``Egref`` and ``dEgdT`` for
+  :func:`~pvlib.pvsystem.calcparams_desoto` (:issue:`462`)
+* Implement a reliable "gold" implementation of the single diode model (SDM)
+  using a bisection method (Brent, 1973) bounded by points known to include the
+  full forward-bias 1st quadrant IV-curve. Also implement a "fast" method using
+  the Newton-Raphson method that is not bounded, but should be safe for well
+  behaved IV-curves. (:issue:`408`)
+* Implement :func:`~pvlib.pvsystem.bishop88` for explicit calculation of
+  arbitrary IV curve points using diode voltage instead of cell voltage. This
+  method is called for ``ivcurve_pnts`` in :func:`~pvlib.pvsystem.singlediode`
```

Comment:
Instead of

> This method is called for ``ivcurve_pnts`` in :func:`~pvlib.pvsystem.singlediode` if the method is either ``'newton'`` or ``'brentq'``, but the IV curve points will be log spaced instead of linear.

If `method` is either ``'newton'`` or ``'brentq'`` and ``ivcurve_pnts`` in :func:`~pvlib.pvsystem.singlediode` is provided, the IV curve points will be log spaced instead of linear.